dangerfile.js

```diff
@@ -75,6 +76,30 @@ function addPercent(change, goodEmoji = '', badEmoji = ':small_red_triangle:') {
   return `+${formatted}% ${badEmoji}`;
 }
 
+function formatDiff(absoluteChange, relativeChange) {
+  if (absoluteChange === 0) {
```

Comment:
Should we have a tolerance for small changes? For instance, in the current report, I notice that `TrapFocus` bundle size has increased by 1 B. `useMediaQuery` bundle size has decreased by 1 B. Maybe we could start with a small threshold of 10 B and see how it behaves from here?

Reply:
Makes sense.

Reply:
Hm thinking about this I don't see a reason for it. It's a change. Negligible but a change. Don't see a reason to hide those.

Reply:
The motivation is about the noise. Do we need to be able to pay attention to small changes?

Reply:
The table does already list everything because it's also about current size. No code can help you if you get anxiety from a 1B change.

Reply:
I like this concept: 

> Information Anxiety is produced by the ever-widening gap between what we understand and what we think we should understand. It is the black hole between data and knowledge, and it happens when information doesn't tell us what we want or need to know.

Anyway, it's not a problem until we experienced it. It's the first time we have +100 items in the table, and growing. We will soo how that goes in practice.